docs/platform/concepts/carbon-footprint.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Carbon footprint
+early: true
+---
+
+import EarlyBadge from "@site/src/components/Badges/EarlyBadge";
```

Comment:
This import doesn't seem to be used?

```suggestion
```

Reply:
Ah, good catch! I must have changed my mind or something... 😅